packages/ts-migrate-plugins/tests/src/explicit-any.test.ts

```diff
@@ -1,26 +1,26 @@
 import ts from 'typescript';
-import { mockPluginParams, mockDiagnostic } from '../test-utils';
+import { mockPluginParams, mockDiagnostic, realPluginParams } from '../test-utils';
 import explicitAnyPlugin from '../../src/plugins/explicit-any';
 
 describe('explicit-any plugin', () => {
   it('adds explicit any', async () => {
-    const text = `import specifier1 from '../source';
+    const text = `let somePromise: any;
 somePromise.then(res1 => res1.default || res1);
 somePromise.then((res2) => res2.default || res2);
+let someArray: any;
 someArray.forEach(({ arg1, arg2 }) => {});
-const { dest1, dest2 } = obj;
 function fn1(p1, p2) {}
 const fn2 = function(p3, p4) {}
-const var1 = [];
-function fn3(p5 = {}) {const {arg3} = p5}
```

Comment:
This is intended to test TS2459, which according to the explicit-any source is supposed to be ["Type '{0}' has no property '{1}' and no string index signatures"](https://github.com/airbnb/ts-migrate/blob/92c465365ab9330658c235697cf29a5d4cbaabb0/packages/ts-migrate-plugins/src/plugins/explicit-any.ts#L169). However, this error text [does not match](https://github.com/microsoft/TypeScript/blob/v4.1.2/src/compiler/diagnosticMessages.json#L1844-L1847) the TypeScript source, which uses that number for a completely different error. There does not appear to be any error message similar to what explicit-any quotes in that entire file.

[TS playground link](https://www.typescriptlang.org/play?module=5&ts=4.1.2#code/GYVwdgxgLglg9mABMMBmAFABwKyILyIDeAvgJREQIDOURAhgE4Dmqx+iOxQA)